docs/docs/gatsby-cli.md

```diff
@@ -233,4 +235,4 @@ In it you’re going to see something like this.
 }
 ```
 
-Edit your `packageManager` value, save and you’re good to go for your next project using `gatsby new`.
+अपना `packageManager` एडिट करें, सहेजें और `Gatsby new` का उपयोग करते हुए आप अपनी अगली परियोजना के लिए जाने के लिए अच्छे हैं
```

Comment:
```
अपना `packageManager` एडिट करें, सहेजें और `Gatsby new` का उपयोग करते हुए आप अपनी अगली परियोजना के लिए जाने के लिए अच्छे हैं
```
 => 
```
अपने `packageManager` की वैल्यू को एडिट करें, सेव करें और `gatsby new` को इस्तेमाल करके आप अपने अगले प्रोजेक्ट के लिए तैयार हैं। 
```

let me know your thoughts on this too!

Reply:
my suggestion..
```अपने `packageManager` की वैल्यू को एडिट करें, सेव करें और `gatsby new` को इस्तेमाल करके आप अपना अगला प्रोजेक्ट बनाने के लिए तैयार हैं।```

Reply:
sounds good